cloud/rackspace/rax_facts.py

```diff
@@ -98,6 +98,11 @@ def rax_facts(module, address, name, server_id):
         except Exception, e:
             pass
 
+    #rackspace likes to leave servers in the deleted state for a while.  
```

Comment:
I actually added these comments to be more clear about why I was doing this.  Do you think it is worth even having them?

I could also shorten it to something like:
# Preventing deleted servers from being included in results